src/test/java/com/desiato/puresynth/BaseTest.java

```diff
@@ -0,0 +1,35 @@
+package com.desiato.puresynth;
+
+import com.desiato.puresynth.repositories.UserRepository;
+import com.desiato.puresynth.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.test.web.servlet.MockMvc;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class BaseTest {
+
+    @Autowired
+    public MockMvc mockMvc;
+
+    @Autowired
+    public UserService userService;
+
+//    @Autowired
+//    public SessionRepository sessionRepository;
```

Comment:
Fields used later should be added in an PR, which will use it. Breaking PR into smaller ones is to make the review process easier. In the PR you should only add what's needed - a minimal set of changes making sense. Adding not existing fields doesn't bring any value:
* the reviewer might have no idea what it is
* it's not used or run - we can't test it
* you can't distribute PRs for review - probably only 1-2 people in the team might now that you're adding `SessionRepository` soon